yt/visualization/tests/test_offaxisprojection.py

```diff
@@ -210,6 +211,35 @@ def test_field_cut_off_axis_octree():
     assert_equal(np.nanmin(p4rho[p4rho > 0.0]) >= 0.5, True)
 
 
+def test_off_axis_octree():
+    ds = fake_octree_ds()
+    p1 = ProjectionPlot(
+        ds,
+        "x",
+        ("gas", "density"),
+        center=[0.6] * 3,
+        width=0.8,
+        weight_field=("gas", "density"),
+    )
+    p2 = OffAxisProjectionPlot(
+        ds,
+        [1, 0, 0],
+        ("gas", "density"),
+        center=[0.6] * 3,
+        width=0.8,
+        weight_field=("gas", "density"),
+    )
+
+    # Note: due to our implementation, the off-axis projection will have a
+    # slightly blurred cell edges so we can't do an exact comparison
+    v1, v2 = p1.frb["gas", "density"], p2.frb["gas", "density"]
+    diff = (v1 - v2) / (v1 + v2) * 2
```

Comment:
Would you mind re-writing this to make it a little clearer on a quick glance that you're comparing the difference to the mean of the two values (unless there's a computational reason that I'm missing)? took me longer than it should have to mentally parse it (though that might be due to a caffeine deficit).

i.e.,:

```python
diff = (v1 - v2) / (( v1 + v2) / 2.0)
```

also, regardless of that, it would be good to include an absolute value as a large negative error would pass the assert test on the following line.